walletlibrary/src/main/java/com/microsoft/walletlibrary/identifier/EncryptedSharedPreferencesIdentifier.kt

```diff
@@ -13,13 +14,14 @@ internal class EncryptedSharedPreferencesIdentifier(
     override val algorithm: String,
     override val method: String,
     override val keyReference: String,
-    private val cryptoOperations: CryptoOperations,
     private val keyStore: EncryptedKeyStore
 ) : HolderIdentifier {
 
-    override fun sign(data: String): String {
-        val keyId = "$id#$keyReference"
-        val privateKey = keyStore.getKey(keyReference)
-        return cryptoOperations.sign(data, privateKey, algorithm, keyId)
+    var jwsHeader: JWSHeader? = null
```

Comment:
Is the jwsHeader always null? What if we use the algorithm reference in the Identifier to construct the proper JwsHeader needed?